content/docs/data_pipeline/introduction/_index.md

```diff
@@ -0,0 +1,77 @@
+---
+weight: 1
+title: "Introduction"
+---
+
+# Introduction
+
+**This is work in progress.**
+
+The FAIR Data Pipeline (FDP) is intended to enable tracking of provenance of data used in epidemiological modelling. File APIs written in C++, Java, Julia, Python and R can be called by model software for data insgestion. These interact with a local relational database storing metadata and the local filesystem, and are configured by a `yml` file associated with the model run. Local files and metadata are synchronised with a remote registry via a command line tool (`fdp`).
```

Comment:
This seems fine on the C++ side, just the "insgestion" typo I noticed.

Reply:
Fixed, thanks!